client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx

```diff
@@ -159,6 +159,30 @@ export const OutboundLineEdit: React.FC<ItemDetailsModalProps> = ({
   const okNextDisabled =
     (mode === ModalMode.Update && nextDisabled) || !currentItem;
 
+  const handleClick = async () => {
+    const allocated = draftStockOutLines?.reduce(
+      (accumulator, stockOutLine) => accumulator + stockOutLine.numberOfPacks,
+      0
+    );
+    if (allocated === 0) {
+      const warningSnack = warning(t('warning.no-quantity-allocated'));
+      warningSnack();
+      return;
+    } else {
```

Comment:
You don't need the `else` since you have an early return. (i.e. it'll only continue to this block if it didn't return from the allocation check)